src/main/kotlin/uk/gov/justice/digital/hmpps/accreditedprogrammesmanageanddeliverapi/api/model/PniScore.kt

```diff
@@ -10,4 +11,18 @@ data class PniScore(
   val overallIntensity: OverallIntensity,
   @Schema(description = "Detailed scores across different assessment domains")
   val domainScores: DomainScores,
+
+  @Schema(
+    example = "  \"riskScores\": {\n" +
+      "    \"ogrs3\": 15.0,\n" +
+      "    \"ovp\": 15.0,\n" +
+      "    \"ospDc\": 1.07,\n" +
+      "    \"ospIic\": 0.11,\n" +
+      "    \"rsr\": 1.46,\n" +
+      "    \"sara\": \"High\"\n" +
+      "  }\n",
+  )
```

Comment:
We can use Kotlins multi-line string (`"""`) to make this a little more readable :

```suggestion
@Schema(
    example = """
      "riskScores": {
        "ogrs3": 15.0,
        "ovp": 15.0,
        "ospDc": 1.07,
        "ospIic": 0.11,
        "rsr": 1.46,
        "sara": "High"
      }
    """
)
```